config/clients/python/template/src/client/configuration.py.mustache

```diff
@@ -19,8 +19,9 @@ class ClientConfiguration(Configuration):
             authorization_model_id=None,
             ssl_ca_cert=None,
             api_url=None, # TODO: restructure when removing api_scheme/api_host
+            timeout_seconds: int | None = None,
```

Comment:
Naming? Looks like other clients are using miliseconds, should update that.

Reply:
Updated throughout to use milliseconds and convert to seconds to provide a valid value for requests/urllib3